src/Database/Migrations/DicomAdapterContextModelSnapshot.cs

```diff
@@ -95,7 +99,7 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                     b.Property<Guid>("InferenceRequestId")
                         .ValueGeneratedOnAdd()
                         .HasColumnType("TEXT")
-                        .HasDefaultValue(new Guid("b932b86a-564b-4cfb-8590-2e98be3aaaf5"));
+                        .HasDefaultValue(new Guid("14580e42-745d-4b76-a24c-24570d5f6568"));
```

Comment:
hey look, another magic value. 

does it need to match the other magic value? then we likely want it as a constant somewhere so that they stay aligned.

Reply:
This is generated code.